### PR TITLE
bootstrap5 migration

### PR DESCRIPTION
**Pull request by DanielW1987**
As discussed, I have not yet customized the Admin Area. That comes next.

Not too much has changed visually in the frontend.

- Bootstrap and Material Icons are now referenced directly instead of going through WebJars. 
- I removed the custom bootstrap theme that we used as a supplement and added any styles to our CSS.
- Scroll to top works correctly again
- jquery removed

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Move to bootstrap 5 incl. remove of jQuery](https://trello.com/c/d9WUd5T9/202-move-to-bootstrap-5-incl-remove-of-jquery)

**Comment by DanielW1987** [comment]
[Rework Scroll to top](https://trello.com/c/DfVrxqp5/374-rework-scroll-to-top)

File: support/src/main/java/rocks/metaldetector/support/infrastructure/WithSensitiveDataRemover.java [hunk 1]
```diff
@@ -37,17 +37,21 @@ default String removeSensitiveDataFromPayload(String originalMessage) throws Jso
...
+          if (payloadAsMap.containsKey(fieldName)) {
+            payloadAsMap.put(fieldName, REMOVED_FOR_LOGGING_STRING);
+          }
+        });
+
+        payload = mapper.writeValueAsString(payloadAsMap);
+        return prePayload.concat(payload);
+      }
+      catch (Exception e) {
+        return originalMessage;
```
Comment (hedgehog1833) [inline support/src/main/java/rocks/metaldetector/support/infrastructure/WithSensitiveDataRemover.java:53]: Maybe log the exception here?

Comment (hedgehog1833) [reply support/src/main/java/rocks/metaldetector/support/infrastructure/WithSensitiveDataRemover.java:53]: And a test would be nice 💪

**Review by hedgehog1833 — APPROVED** [review]

File: support/src/main/java/rocks/metaldetector/support/infrastructure/WithSensitiveDataRemover.java (hunk 1 above)
Comment (DanielW1987) [reply support/src/main/java/rocks/metaldetector/support/infrastructure/WithSensitiveDataRemover.java:53]: see f202cb2

**Pull request merged**